integration/xealth_test.go

```diff
@@ -667,6 +604,10 @@ func asServer(req *http.Request) {
 	req.Header.Set("x-tidepool-session-token", integrationTest.TestServerToken)
 }
 
+func asServiceAccount(req *http.Request) {
+	req.Header.Set("x-tidepool-session-token", integrationTest.TestServiceAccountToken)
```

Comment:
x-tidepool-session-token exists as both a var and a const in the auth package:

auth/authentication.go:18
auth/authorizer.go:23

It'd probably be best to settle on one usage (an exported const would be my preference).

That said, this code is accurate and working, so not a blocker.

Reply:
An export const won't catch an accidental change of the header key. If we had an exported const in the auth package e.g. `TidepoolSessionTokenKey` and we had both the implementation and tests use the value, we could set the value to an arbitrary string (e.g. `x-session-token`) and tests would still pass. However, this will be a breaking change because existing clients use `x-tidepool-session-token`.